Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -50,41 +52,46 @@ u64 WbfsFileReader::GetDataSize() const
   return WII_SECTOR_COUNT * WII_SECTOR_SIZE;
 }
 
-bool WbfsFileReader::OpenFiles(const std::string& filename)
+void WbfsFileReader::OpenAdditionalFiles(const std::string& path)
 {
-  m_total_files = 0;
+  if (path.length() < 4)
+    return;
 
   while (true)
   {
-    auto new_entry = std::make_unique<file_entry>();
-
     // Replace last character with index (e.g. wbfs = wbf1)
-    std::string path = filename;
-    if (m_total_files != 0)
-    {
-      path[path.length() - 1] = '0' + m_total_files;
-    }
+    std::string current_path = path;
+    if (m_files.size() >= 10)
```

Comment:
I don't know if that's faster. Keep in mind that it will waste some time every time there is an additional file. I would prefer to avoid it unless someone really wants to benchmark it.

@lioncash, what do you think of putting an assert there? (The function works as intended for all values of m_files.size(), except 0, which will make it open a .wbf0 file instead of .wbfs. The old version had an extra if statement for using .wbfs when m_total_files was 0.)